src/inmanta/data/__init__.py

```diff
@@ -2261,6 +2275,14 @@ def to_dto(self) -> m.Environment:
             validator=convert_int,
             doc="The number of days to retain notifications for",
         ),
+        RECOMPILE_BACKOFF: Setting(
+            name=RECOMPILE_BACKOFF,
+            default=0.1,
+            typ="float",
```

Comment:
For this setting only values `>=0` make sense. I think it would make sense to introduce a type `positive_float` and use that one instead.